### DISCUSSION TITLE
キャラクターをVRoid SDK for Unreal Engine 1.2.0の時の外観に戻したい

### DISCUSSION THREAD

**Discussion by papicochan** [pixiv/vroid-sdk-developers#183] [Q&A] [answered]
1.2.0から1.3.0にアップグレードした際に、キャラクターの色味、明るさ、ライティングの影響を受けるかどうかなどの初期設定が変更されたように思うのですが、元の設定に戻すための設定や数値を入れる箇所があれば教えていただけませんでしょうか。

**Comment by mameoPix** [comment, marked as answer]
以下の2つのBlueprint(BP_VRoidMannequinRetargetAttachActor, BP_VRoidMannequinRetargetCharacter)内で
`Construct Vrm Asset List Object BPUE5`を使用していることが原因かと思います。

1.2.0以前は`VrmAssetListObjectBP`を使用しておりましたので
使用するアセットをVrmAssetListObjectBPに変更することで従来の見た目になるかと思います。

VrmAssetListObjectBPはUE4向けのMaterialが設定されているため
UE5向けに設定し直したことが原因で従来のキャラクター表現と異なる見た目になってしまっています。

BP_VRoidMannequinRetargetAttachActor
![image](https://github.com/user-attachments/assets/2106be5e-ba76-4583-bc42-518e747f1c37)
BP_VRoidMannequinRetargetCharacter
![image](https://github.com/user-attachments/assets/d629a771-74ae-481b-8d6c-e888d52195d9)

上記対応を行っても変更が確認できない場合、他にも気になる点があればご報告いただけますと幸いです。

**Reply by papicochan** [reply]
ありがとうございます！！元の見た目に戻すことができました！！
新しい方の見た目が白飛びしてしまうアバターが多いように感じているのでしばらく以前の外観で使用したいと思います。
